test/aries-js-worker/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "aries-js-worker-test",
+  "version": "0.1.0",
```

Comment:
suggest setting private flag to true.

https://docs.npmjs.com/files/package.json#private

Reply:
done